packages/validate/src/index.ts

```diff
@@ -27,15 +28,20 @@ const validateOptions: ValidateOptions = {
 
 async function getLocalSchemaPaths(localSchemasDir: string, options: ValidateOptions): Promise<string[]> {
   const { ignoredPaths = [], ignoredExts = [] } = options;
-  const directories = (await readDir(localSchemasDir))
+  const dirsAndFiles = (await readDir(localSchemasDir))
     .filter((name: string) => !ignoredPaths.includes(name) && ignoredExts.every((ext) => !name.endsWith(ext)))
     .map((dir: string) => path.resolve(localSchemasDir, dir)) as string[];
 
   const subDirectories = await Promise.all(
-    directories.map(async (dir) => {
-      return (await readDir(dir))
+    dirsAndFiles.map(async (dirOrFile) => {
+      const isFile = fs.lstatSync(dirOrFile).isFile();
+      if (isFile) {
+        return dirOrFile;
```

Comment:
Here the file is always an absolute path right ?
Otherwise you might try to resolve it.